src/os/bluestore/BlueFS.cc

```diff
@@ -3240,7 +3233,7 @@ void BlueFS::sync_metadata(bool avoid_compact)
     start = ceph_clock_now();
     *_dout <<  dendl;
     flush_bdev(); // FIXME?
-    _flush_and_sync_log(l, false);
+    _flush_and_sync_log(l);
```

Comment:
you're enforcing incremental update at this point - I'm not sure that's a valid behavior, full metadata snapshot to be created here.

Reply:
ah, nevermind. This produces incremental updates for dirty files only